app/include/zmk/keymap.h

```diff
@@ -17,3 +17,12 @@ int zmk_keymap_layer_deactivate(uint8_t layer);
 int zmk_keymap_layer_toggle(uint8_t layer);
 
 int zmk_keymap_position_state_changed(uint32_t position, bool pressed, int64_t timestamp);
+
+#define KEY_BINDING_TO_STRUCT(idx, drv_inst)                                                       \
```

Comment:
In general, this looks good. Can we name this better though? `ZMK_KEYMAP_EXTRACT_BINDING` maybe?